src/test/CMakeLists.txt

```diff
@@ -573,6 +573,12 @@ if(WITH_MGR)
   list(APPEND tox_tests run-tox-mgr-orchestrator_cli)
   set(MGR_ORCHESTRATOR_CLI_VIRTUALENV ${CEPH_BUILD_VIRTUALENV}/mgr-orchestrator_cli-virtualenv)
   list(APPEND env_vars_for_tox_tests MGR_ORCHESTRATOR_CLI_VIRTUALENV=${MGR_ORCHESTRATOR_CLI_VIRTUALENV})
+
+  add_test(NAME run-tox-ceph-common
```

Comment:
please note, we have a library named ceph-common. and "ceph" is a little bit redundant in this context. could you use another name, like "run-tox-python-common" ?

Reply:
fixed